src/dp_util.c

```diff
@@ -160,3 +166,45 @@ void dp_free_jhash_table(struct rte_hash *table)
 {
 	rte_hash_free(table);
 }
+
+int dp_set_vf_rate_limit(uint16_t port_id, uint64_t rate)
+{
+	char filename[DP_SYSFS_MAX_PATH];
+	uint16_t vf_pattern_len = 0;
+	const char *pattern = dp_conf_get_vf_pattern();
+	FILE *fp;
+	struct dp_port *port = dp_get_port_by_id(port_id);
+	uint64_t rate_in_mbits = rate;
+
+	if (!port) {
+		DPS_LOG_ERR("Cannot get port by id", DP_LOG_PORTID(port_id));
+		return DP_ERROR;
+	}
```

Comment:
No need to log here (unless you want to show the current function) this already logs the unlikely situation when the port is not registered in dpservice.

Reply:
log info is removed